test/controllers/IndexControllerSpec.scala

```diff
@@ -18,278 +18,162 @@ package controllers
-              .set(OldServiceSubmissionPage, OldServiceSubmission("num", "date"))
-              .success
-              .value
-          )
-        )
-      )
-      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future.successful(true))
+    "there are no user-answers and no session id" must {
 
-      lazy val result = controller.onPageLoad()(fakeRequest)
+      "redirect to CannotFindApplication page" in {
 
-      status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual routes.ApplicationBeingProcessedController.onPageLoad.url
-      verify(mockUserAnswerService, times(1)).get(any())(any(), any())
-    }
+        val controller: IndexController = applicationBuilder().injector.instanceOf[IndexController]
 
-    "Redirect to error page if the charitiesSave4LaterService returing error" in {
-      lazy val service = new FakeCharitiesSave4LaterService(
-        mockCharitiesShortLivedCache,
-        mockUserAnswerTransformer,
-        mockSessionRepository,
-        mockUserAnswerService,
-        mockAuditService,
-        None
-      )
-      val app          =
-        new GuiceApplicationBuilder()
-          .overrides(
-            bind[CharitiesSave4LaterService].toInstance(service),
-            bind[UserAnswerService].toInstance(mockUserAnswerService),
-            bind[CharitiesShortLivedCache].toInstance(mockCharitiesShortLivedCache),
-            bind[AuthIdentifierAction].to[FakeAuthIdentifierAction]
-          )
-          .build()
-
-      val controller: IndexController = app.injector.instanceOf[IndexController]
-
-      when(mockCharitiesShortLivedCache.fetchAndGetEntry[Boolean](any(), any())(any(), any(), any()))
-        .thenReturn(Future.successful(None))
-      when(mockUserAnswerService.get(any())(any(), any())).thenReturn(
-        Future.successful(
-          Some(
-            emptyUserAnswers
-              .set(OldServiceSubmissionPage, OldServiceSubmission("num", "date"))
-              .success
-              .value
-          )
-        )
-      )
-      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future.successful(true))
+        when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future(None))
 
-      lazy val result = controller.onPageLoad()(fakeRequest)
+        lazy val result = controller.onPageLoad()(fakeRequest)
 
-      status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual routes.ApplicationBeingProcessedController.onPageLoad.url
-      verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+        status(result) mustEqual 303
+        redirectLocation(result).value mustBe controllers.routes.CannotFindApplicationController.onPageLoad.url
+        verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+
+      }
     }
 
-    "Set answers and redirect to the next page (start of journey page)" in {
+    "there are  empty useranswers and a session id" must {
```

Comment:
Can a space be added for useranswers i.e. `user answers` like for session id. Thanks